internal/blockchain/ethereum/quorum/quorum_provider.go

```diff
@@ -236,43 +237,46 @@ func (p *QuorumProvider) DeployFireFlyContract() (*types.ContractDeploymentResul
 	return p.connector.DeployContract(contract, "FireFly", p.stack.Members[0], nil)
 }
 
+func (p *QuorumProvider) buildTesseraServiceDefinition(memberIdx int) *docker.ServiceDefinition {
```

Comment:
only moved this out as it was a big block of in-lined code - no functional change